lib/src/cloud_firestore_mocks_base.dart

```diff
@@ -85,7 +85,7 @@ class MockFirestoreInstance extends Mock implements FirebaseFirestore {
       handlerResult.values.forEach(_validateTransactionReturnValue);
```

Comment:
Since https://github.com/FirebaseExtended/flutterfire/issues/1642 is now fixed. Is it safe to remove this loop entirely?

Reply:
If so, then the function can `return transactionHandler(transaction);` directly.

Reply:
oh... thanks 👍 
we can remove workaround code. So, I removed https://github.com/atn832/cloud_firestore_mocks/pull/130/commits/cbe0d656c4cb844da1a93391fb590cdba5cdc092